hudi-cli/src/main/java/org/apache/hudi/cli/commands/LockCommand.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.cli.commands;
+
+import org.apache.hudi.cli.HoodieCLI;
+import org.apache.hudi.common.table.HoodieTableMetaClient;
+import org.apache.hudi.storage.StoragePath;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+import org.springframework.shell.standard.ShellOption;
+
+import org.apache.hudi.common.util.FileIOUtils;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import static org.apache.hudi.client.transaction.lock.audit.StorageLockProviderAuditService.AUDIT_FOLDER_NAME;
+
+/**
+ * CLI commands for managing Hudi table locking and audit functionality.
+ */
+@ShellComponent
+public class LockCommand {
+
+  private static final Logger LOG = LoggerFactory.getLogger(LockCommand.class);
+  private static final String AUDIT_CONFIG_FILE_NAME = "audit_enabled.json";
```

Comment:
Could `AuditServiceFactory.AUDIT_CONFIG_FILE_NAME` be reused?

Reply:
moved everything to shared config file